weeklyChallenges01/ChallengesWithTest01.cs

```diff
@@ -8,37 +8,59 @@ public class ChallengesSet01
     {
         public bool AreTwoNumbersTheSame(int num1, int num2)
         {
-            throw new NotImplementedException();
+            if (num1 == num2)
+            {
+                
+                return true;
+            }
+            else
+            {
+               
+                return false;
+            }
         }
 
         public double Subtract(double minuend, double subtrahend)
         {
-            throw new NotImplementedException();
+            double total;
+            total = minuend - subtrahend;
```

Comment:
creating a variable is fine but for optimization consider simply putting `return minuend - subtrahend;` it will give the same result, using fewer lines.